lib/active_fedora/core.rb

```diff
@@ -62,6 +62,13 @@ def reload
       refresh
       load_attached_files
       self
+    rescue Ldp::Gone => err
+      logger.info("Tried to reload an object that has been destroyed; " \
+                        "this is probably due to mismanagement of the object's " \
```

Comment:
I would think this would most likely be the result of having two separate instances and destroying one of them and calling reload on the other.  I'm not sure that this conjecture is helpful.